### PR TITLE
Fixed #743 -- Added past donations to "manage donations" page.

### LINKED ISSUE #743: Add past donations to "manage donations" page
As requested in https://github.com/django/djangoproject.com/issues/742, the `/fundraising/manage-donations/<donor-id>/` view could list past donations.

### PR DESCRIPTION
**Pull request by chaitu210**
Added one time donations by filtering the DjangoHero donations with stripe_subscription_id and interval as onetime.

### CODE REVIEW

File: fundraising/views.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ def thank_you(request, donation):
 def manage_donations(request, hero):
     hero = get_object_or_404(DjangoHero, pk=hero)
     recurring_donations = hero.donation_set.exclude(stripe_subscription_id='')
+    onetime_donations = hero.donation_set.filter(stripe_subscription_id='', interval="onetime")
```
Comment (timgraham) [inline fundraising/views.py]: Use single quotes.

File: djangoproject/templates/fundraising/manage-donations.html [hunk 2]
```diff
@@ -65,6 +65,17 @@
   {% else %}
 </form>
   {% endif %}
+  {% if onetime_donations %}
+  <hr />
+    <h2>Your past One-time donations</h2>
+    <ul>
+    {% for donation in onetime_donations %}
+      <li>
+        You have donated ${{ donation.subscription_amount }} on {{ donation.created|date:"N jS, Y \a\t P" }}
```
Comment (timgraham) [inline djangoproject/templates/fundraising/manage-donations.html]: I don't "You have donated" isn't needed on each line.

File: fundraising/tests/test_models.py [hunk 3]
```diff
@@ -69,3 +69,25 @@ class TestInKindDonor(TestCase):
     def test_display_name(self):
         donor = InKindDonor(name='Hero')
         self.assertEqual(donor.display_name, 'Hero')
+
+
+class TestOnetimeDonationDjangoHero(TestCase):
```
Comment (timgraham) [inline fundraising/tests/test_models.py]: The test doesn't correspond to the new functionality. You should test the `manage_donations` view by checking the new context variable and its rendering in the template.

Comment (chaitu210) [reply fundraising/tests/test_models.py]: @timgraham pushed the code with the updated test case. Can you please review it?

**Comment by timgraham** [comment]
How about if we display all past donations, not just the one-time payments? I amended the PR.

**Comment by chaitu210** [comment]
@timgraham sure I'll look into the code add a new pull request to display past donations.

**Pull request merged**